packages/meditrak-server/src/routes/GETHandler/helpers.js

```diff
@@ -103,7 +104,7 @@ export const getQueryOptionsForColumns = (
           `${recordType}.id`,
           getForeignKeyColumnName(recordType),
         ];
-        multiJoin.push({ joinWith: recordType, joinCondition });
+        multiJoin.push({ joinWith: recordType, joinCondition, joinType: defaultJoinType });
```

Comment:
So if `joinType` is null, it will use the original default (which I guess is inner)?

Reply:
Yep! Made sure to double check that